pkg/config/config.go

```diff
@@ -191,6 +191,10 @@ type EngineConfig struct {
 	// container images.  Valid values are "oci" (default) or "docker".
 	ImageBuildFormat string `toml:"image_build_format,omitempty"`
 
+	// ImageParallelPulls indicates the maximum number of layers of an
+	// image to be pulled simultaneously.
+	ImageParallelPulls uint `toml:"image_parallel_pulls,omitempty"`
```

Comment:
The `0` value needs to either be rejected (maybe, if option presence has meaning), or documented.

Reply:
I documented it.